gtest/src/program.rs

```diff
@@ -265,4 +264,39 @@ mod tests {
         assert!(!run_result.main_failed());
         assert!(run_result.log.contains(&expected_log));
     }
+
+    #[test]
+    fn mailbox_mock_works(){
+        //Arrange
+        let system = System::new();
+        let message_id: MessageId = Default::default();
+        let source_user_id = ProgramIdWrapper::from(100).0;
+        let destination_user_id = ProgramIdWrapper::from(200).0;
+        let message_payload: Payload = vec![1,2,3].into();
+        let reply_payload: Payload = vec![3,2,1].into();
+        let program = manager::Actor::Mock(Box::new(()));//should put some wasm program into box
```

Comment:
You don't need the actor to test it. You can check mailbox interaction between two users